{"name": "Проверка раздела «Лента заказов»", "status": "broken", "statusDetails": {"message": "ValueError: response body:\n{\"message\":\"API rate limit exceeded for 5.199.209.205. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)\",\"documentation_url\":\"https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting\"}\n\nrequest url:\nhttps://api.github.com/repos/mozilla/geckodriver/releases/latest\nresponse headers:\n{'Date': 'Fri, 28 Jun 2024 12:55:10 GMT', 'Server': 'Varnish', 'Strict-Transport-Security': 'max-age=31536000; includeSubdomains; preload', 'X-Content-Type-Options': 'nosniff', 'X-Frame-Options': 'deny', 'X-XSS-Protection': '1; mode=block', 'Content-Security-Policy': \"default-src 'none'; style-src 'unsafe-inline'\", 'Access-Control-Allow-Origin': '*', 'Access-Control-Expose-Headers': 'ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-RateLimit-Used, X-RateLimit-Resource, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset', 'Content-Type': 'application/json; charset=utf-8', 'Referrer-Policy': 'origin-when-cross-origin, strict-origin-when-cross-origin', 'X-GitHub-Media-Type': 'github.v3; format=json', 'X-RateLimit-Limit': '60', 'X-RateLimit-Remaining': '0', 'X-RateLimit-Reset': '1719579766', 'X-RateLimit-Resource': 'core', 'X-RateLimit-Used': '60', 'Content-Length': '279', 'X-GitHub-Request-Id': 'F5A6:A1617:4900CF6:49BFA1F:667EB2AE'}", "trace": "conftest.py:27: in driver\n    driver = _get_driver(request.param)\n        request    = <SubRequest 'driver' for <Function test_order_number_in_section_in_work[firefox]>>\nconftest.py:19: in _get_driver\n    service = FirefoxService(executable_path=GeckoDriverManager().install())\n        name       = 'firefox'\nC:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\webdriver_manager\\firefox.py:39: in install\n    driver_path = self._get_driver_binary_path(self.driver)\n        self       = <webdriver_manager.firefox.GeckoDriverManager object at 0x0000020020ABA7B0>\nC:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\webdriver_manager\\core\\manager.py:35: in _get_driver_binary_path\n    binary_path = self._cache_manager.find_driver(driver)\n        driver     = <webdriver_manager.drivers.firefox.GeckoDriver object at 0x0000020020ABAD20>\n        self       = <webdriver_manager.firefox.GeckoDriverManager object at 0x0000020020ABA7B0>\nC:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\webdriver_manager\\core\\driver_cache.py:105: in find_driver\n    driver_version = self.get_cache_key_driver_version(driver)\n        browser_type = 'firefox'\n        browser_version = '125.0'\n        driver     = <webdriver_manager.drivers.firefox.GeckoDriver object at 0x0000020020ABAD20>\n        driver_name = 'geckodriver'\n        os_type    = 'win64'\n        self       = <webdriver_manager.core.driver_cache.DriverCacheManager object at 0x0000020020ABA930>\nC:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\webdriver_manager\\core\\driver_cache.py:152: in get_cache_key_driver_version\n    return driver.get_driver_version_to_download()\n        driver     = <webdriver_manager.drivers.firefox.GeckoDriver object at 0x0000020020ABAD20>\n        self       = <webdriver_manager.core.driver_cache.DriverCacheManager object at 0x0000020020ABA930>\nC:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\webdriver_manager\\core\\driver.py:48: in get_driver_version_to_download\n    return self.get_latest_release_version()\n        self       = <webdriver_manager.drivers.firefox.GeckoDriver object at 0x0000020020ABAD20>\nC:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\webdriver_manager\\drivers\\firefox.py:29: in get_latest_release_version\n    resp = self._http_client.get(\n        determined_browser_version = '125.0'\n        self       = <webdriver_manager.drivers.firefox.GeckoDriver object at 0x0000020020ABAD20>\nC:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\webdriver_manager\\core\\http.py:36: in get\n    self.validate_response(resp)\n        kwargs     = {'headers': None}\n        resp       = <Response [403]>\n        self       = <webdriver_manager.core.http.WDMHttpClient object at 0x0000020020ABAD50>\n        url        = 'https://api.github.com/repos/mozilla/geckodriver/releases/latest'\nC:\\Users\\ilme5\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\webdriver_manager\\core\\http.py:19: in validate_response\n    raise ValueError(\nE   ValueError: response body:\nE   {\"message\":\"API rate limit exceeded for 5.199.209.205. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)\",\"documentation_url\":\"https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting\"}\nE   \nE   request url:\nE   https://api.github.com/repos/mozilla/geckodriver/releases/latest\nE   response headers:\nE   {'Date': 'Fri, 28 Jun 2024 12:55:10 GMT', 'Server': 'Varnish', 'Strict-Transport-Security': 'max-age=31536000; includeSubdomains; preload', 'X-Content-Type-Options': 'nosniff', 'X-Frame-Options': 'deny', 'X-XSS-Protection': '1; mode=block', 'Content-Security-Policy': \"default-src 'none'; style-src 'unsafe-inline'\", 'Access-Control-Allow-Origin': '*', 'Access-Control-Expose-Headers': 'ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-RateLimit-Used, X-RateLimit-Resource, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset', 'Content-Type': 'application/json; charset=utf-8', 'Referrer-Policy': 'origin-when-cross-origin, strict-origin-when-cross-origin', 'X-GitHub-Media-Type': 'github.v3; format=json', 'X-RateLimit-Limit': '60', 'X-RateLimit-Remaining': '0', 'X-RateLimit-Reset': '1719579766', 'X-RateLimit-Resource': 'core', 'X-RateLimit-Used': '60', 'Content-Length': '279', 'X-GitHub-Request-Id': 'F5A6:A1617:4900CF6:49BFA1F:667EB2AE'}\n        resp       = <Response [403]>\n        status_code = 403"}, "description": "Проверка что после оформления заказа его номер появляется в разделе В работе", "attachments": [{"name": "log", "source": "d2ef1f99-775b-4f27-bb58-8c8a2ccbc1d2-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1719579310964, "stop": 1719579310964, "uuid": "1930d09f-8f2e-438a-a623-4f767787d710", "historyId": "46e2ca8a81dc69f3faf9acb5a3fed741", "testCaseId": "81eb7fa9d6a21190c264dc92f3c40297", "fullName": "tests.test_order_page.TestOrderFeed#test_order_number_in_section_in_work", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_page"}, {"name": "subSuite", "value": "TestOrderFeed"}, {"name": "host", "value": "DESKTOP-8VJF26I"}, {"name": "thread", "value": "11160-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_page"}]}